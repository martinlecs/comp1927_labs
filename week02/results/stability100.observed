
1 aba
1 aab
2 aba
2 aab
3 aba
3 aab
4 aba
4 aab
5 aab
5 aba
6 aab
6 aba
7 aba
7 aab
8 aba
8 aab
9 aba
9 aab
10 aab
10 aba
11 aab
11 aba
12 aab
12 aba
13 aab
13 aba
14 aab
14 aba
15 aba
15 aab
16 aab
16 aba
17 aba
17 aab
18 aba
18 aab
19 aba
19 aab
20 aba
20 aab
21 aab
21 aba
22 aab
22 aba
23 aab
23 aba
24 aab
24 aba
25 aab
25 aba
26 aab
26 aba
27 aba
27 aab
28 aba
28 aab
29 aba
29 aab
30 aba
30 aab
31 aba
31 aab
32 aba
32 aab
33 aba
33 aab
34 aab
34 aba
35 aab
35 aba
36 aab
36 aba
37 aab
37 aba
38 aba
38 aab
39 aba
39 aab
40 aab
40 aba
41 aba
41 aab
42 aba
42 aab
43 aba
43 aab
44 aba
44 aab
45 aba
45 aab
46 aba
46 aab
47 aab
47 aba
48 aba
48 aab
49 aab
49 aba
50 aab
50 aba
