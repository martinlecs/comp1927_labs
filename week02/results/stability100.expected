
1 aab
1 aba
2 aab
2 aba
3 aab
3 aba
4 aab
4 aba
5 aab
5 aba
6 aab
6 aba
7 aab
7 aba
8 aab
8 aba
9 aab
9 aba
10 aab
10 aba
11 aab
11 aba
12 aab
12 aba
13 aab
13 aba
14 aab
14 aba
15 aab
15 aba
16 aab
16 aba
17 aab
17 aba
18 aab
18 aba
19 aab
19 aba
20 aab
20 aba
21 aab
21 aba
22 aab
22 aba
23 aab
23 aba
24 aab
24 aba
25 aab
25 aba
26 aab
26 aba
27 aab
27 aba
28 aab
28 aba
29 aab
29 aba
30 aab
30 aba
31 aab
31 aba
32 aab
32 aba
33 aab
33 aba
34 aab
34 aba
35 aab
35 aba
36 aab
36 aba
37 aab
37 aba
38 aab
38 aba
39 aab
39 aba
40 aab
40 aba
41 aab
41 aba
42 aab
42 aba
43 aab
43 aba
44 aab
44 aba
45 aab
45 aba
46 aab
46 aba
47 aab
47 aba
48 aab
48 aba
49 aab
49 aba
50 aab
50 aba
